packages/server/graphql/public/types/RemoveIntegrationSearchQuerySuccess.ts

```diff
@@ -0,0 +1,18 @@
+export type RemoveIntegrationSearchQuerySuccessSource =
+  | {
+      userId: string
+      teamId: string
+    }
+  | {error: {message: string}}
+
+const RemoveIntegrationSearchQuerySuccess: {
+  jiraServerIntegration: (
+    source: RemoveIntegrationSearchQuerySuccessSource
+  ) => RemoveIntegrationSearchQuerySuccessSource
+} = {
+  jiraServerIntegration: (source) => {
```

Comment:
+1 did you test this resolver is actually necessary? Also you would usually import the generated types here
```
const RemoveIntegrationSearchQuerySuccess: RemoveIntegrationSearchQuerySuccessResolvers = {
```
You might need to define the `JiraServerIntegration` source properly though.

Reply:
@Dschoordsch Does not work for me without this resolver, `jiraServerIntegration` returned as `null.
Picking generated type `RemoveIntegrationSearchQuerySuccessResolvers` does not work in this case as I need to just forward plain `source` here, not actually resolve.